### PR TITLE
GCP Storage, AWS S3, OpenFaaS, RabbitMQ all new output types values added

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

> If this PR will release a new chart version please make sure to also uncomment the following line:

/kind chart-release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area falco-chart

> /area falco-exporter-chart

/area falcosidekick-chart

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] CHANGELOG.md updated

### CODE REVIEW

**Comment by leogr** [comment]
Hey @developer-guy 

Thank you for your contribution! :+1: 

Could you bump the falcosidekick version? It's needed to pass the CI check but also to release the chart with your modifications.

**Comment by developer-guy** [comment]
should I do upgrade it to 0.2.10, right? @leodido

**Comment by developer-guy** [comment]
> should I do upgrade it to 0.2.10, right? @leogr

**Comment by leogr** [comment]
> should I do upgrade it to 0.2.10, right? @leogr or is this a minor type of change because we are adding a new feature into the project?

I'd go for 0.3.0 because of the new features, but maybe @Issif has a better opinion than mine?

PS don't forget to update the changelog too, pls

File: falcosidekick/README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Currently available outputs are :
 * [**Apache Kafka**](https://kafka.apache.org/)
 * [**PagerDuty**](https://pagerduty.com/)
 * [**Kubeless**](https://kubeless.io/)
-* [**Kubeless**](https://kubeless.io/)
+* [**OpenFaaS**](https://www.openfaas.com/)
```
Comment (Issif) [inline falcosidekick/README.md:47]: Can you also add new outputs: RabbitMQ, GCP Storage, AWS S3 (see Readme of falcosidekick)

**Review by Issif — CHANGES_REQUESTED** [review]
can you bump the version of `falcosidekick-ui` image to `v1.0.0` please and update the changelog like @leogr mentionned, please 🙏

File: falcosidekick/README.md (hunk 1 above)
Comment (developer-guy) [reply falcosidekick/README.md:47]: Okay, let me update it.

**Comment by developer-guy** [comment]
> can you bump the version of `falcosidekick-ui` image to `v1.0.0` please and update the changelog like @leogr mentionned, please 🙏

It should be finished, would you like to review it again @lssif ?

File: falcosidekick/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,28 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 1.0.0
```
Comment (Issif) [inline falcosidekick/CHANGELOG.md:8]: I wasn't clear, sorry.

For this chart, ok for bumping to 0.3.0. 

But, we also need to bump up the version of `falcosidekick-ui` to `v1.0.0`: https://github.com/falcosecurity/charts/blob/b52c1914b4731fc65fe5c5d515d712aa303c8d08/falcosidekick/values.yaml#L248

and you can increase the retention to 200 too: https://github.com/falcosecurity/charts/blob/b52c1914b4731fc65fe5c5d515d712aa303c8d08/falcosidekick/values.yaml#L255

Comment (developer-guy) [reply falcosidekick/CHANGELOG.md:8]: no worries, it is my mistake, I couldn't understand it correctly 🤦

Comment (developer-guy) [reply falcosidekick/CHANGELOG.md]: Okay, it is done now.

File: falcosidekick/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,28 @@ numbering uses [semantic versioning](http://semver.org).
...
+* Use higher level Writer api for Kafka
+* Reorder imports to follow good practices
+* Prevent misleading error message when CUSTOMFIELDS env var is set
+* Use Events v2 API for PagerDUty output
+
+### Fixes
+
+* Fix outputformat when using fields or text in Slack output
+* Fix HTML template for SMTP output
+
```
Comment (Issif) [inline falcosidekick/CHANGELOG.md:28]: the changes are related to `falcosidekick` itself, not the chart, you can remove them

**Review by Issif — CHANGES_REQUESTED** [review]
you forgot to update the image of `falcosidekick` to `2.22.0` ^^

File: falcosidekick/CHANGELOG.md (hunk 3 above)
Comment (developer-guy) [reply falcosidekick/CHANGELOG.md]: Okay

**Review by Issif — APPROVED** [review]

**Pull request merged**